src/magnetometer.rs

```diff
@@ -54,6 +54,25 @@ where
             z: data.2 as i16,
         })
     }
+
+    /// Enable or disable the magnetometer's built in offset cancellation
+    ///
+    /// Offset cancellation is **automatically** managed by the device in **continuous** mode.
+    ///
+    /// Offset cancellation has to be **managed by the user** in **single measurement** (OneShot) mode averaging
+    /// two consecutive measurements H<sub>n</sub> and H<sub>n-1</sub>.
+    pub fn mag_offset_cancellation(&mut self, enable: bool) -> Result<(), Error<CommE, PinE>> {
```

Comment:
Could you rather define a method to enable offset calculation and another one to disable it?
Something like `enable_mag_offset_cancellation` and `disable_mag_offset_cancellation`.
(same below)